shell/platform/fuchsia/flutter/tests/integration/touch-input/embedding-flutter-view/lib/embedding-flutter-view.dart

```diff
@@ -173,3 +245,14 @@ ViewHolderToken _launchGfxChildView() {
 
   return viewHolderToken;
 }
+
+List<String> _GetArgsFromConfigFile() {
+  List<String> args;
+  final f = File(_argsCsvFilePath);
+  if (!f.existsSync()) {
+    return List.empty();
+  }
+  final fileContentCsv = f.readAsStringSync();
+  args = fileContentCsv.split('\n');
+  return args;
+}
```

Comment:
No need to change anything here, just curious -- is structured component configuration available in flutter yet? If so, for future reference, this might be a good use case for it :) https://fuchsia.dev/fuchsia-src/development/components/configuration/structured_config

Reply:
I'm not sure what the status of this for OOT use is :( but I can test it out and let you know

Reply:
Don't worry about it for this change, just a thought for future reference if you ever find yourself manually constructing something similar.